certif/tests/acceptance/session-details_test.js

```diff
@@ -74,18 +74,15 @@ module('Acceptance | Session Details', function (hooks) {
       assert.equal(currentURL(), '/sessions/liste');
     });
 
-    test('it should show the number of candidates on tab', async function (assert) {
+    test.only('it should show the number of candidates on tab', async function (assert) {
```

Comment:
😏

Reply:
Et la CI ne sort pas en erreur :thinking:

Reply:
ohoh , on est sur qunit et pas Mocha
Time to leverage https://github.com/platinumazure/eslint-plugin-qunit ?

Reply:
:duck: needed - @ajoanny ?